app/views/jobseekers/alert_mailer/alert.text.erb

```diff
@@ -2,42 +2,24 @@
 
 ---
 <%- @vacancies.each do |vacancy| %>
-  <%= show_link(vacancy) %>
-  <%= vacancy_location_with_organisation_link(vacancy) %>
-
-  <%= t(".salary", salary: vacancy.salary) %>
+  <%= vacancy.job_title %>
+  <%= vacancy_organisation_and_location(vacancy) %>
   <%= t(".working_pattern", working_pattern: VacancyPresenter.new(vacancy).readable_working_patterns_with_details) %>
-  <%= t(".closing_date", closing_date: format_time_to_datetime_at(vacancy.expires_at)) %>
-
-  <%= '^ ' + t(".quick_apply") if vacancy.enable_job_applications? %>
-
+  <%= show_link(vacancy) %>
   ---
 <% end %>
 
 # <%= t(".title") %>
-<%= t("subscriptions.intro") %>
 
 <%- subscription.filtered_search_criteria.each_pair do |filter, value| %>
   <%= "- #{filter.humanize}: #{value}" %>
 <% end %>
 
-<%= t(".alert_frequency", frequency: subscription.frequency) %>
+<%- if jobseeker_has_profile? %>
+  
+<% end %>
```

Comment:
Not sure is this is a work in progress, but this seems empty? :thinking:

Reply:
Yep, fixed now. Thanks Marc